### ISSUE TITLE
fail2ban email notification

### ISSUE THREAD

**Issue by clown522** [nextcloud/nextcloudpi#540]
Hi @nachoparker,
i do not want to annoy you but is it possible that the email notification of fail2ban does not sent any mail?
fail2ban works now also at your web interface but without mailalert. (v 0.56.21)
Could you test it when you have time?
Thanks for fixing problems so fast!

**Comment by nachoparker** [comment]
it worked last time I checked it, I'll see if I have time to test it again.

or can anybody confirm that it works? (I have it disabled)

**Comment by nachoparker** [comment]
Hi, @crossr1 kindly tried to reproduce this issue, and he was not able to.

fail2ban email notifications work fine for him. I will close this now. Please, reopen with details on how to reproduce if the problem still persists.

Thaks for your feedback

**Comment by sonicwave2020** [comment]
Had these logs in mail log. 
/etc/aliases.db no such file or directory

**Comment by nachoparker** [comment]
thanks for reporting.

Maybe @crossr1 or @ovpc can confirm whether they also have that `aliases.db` error in their logs.

Have you checked fail2ban logs?

**Comment by ovpc** [comment]
No aliases.db error here  (curl install on Debian9-NUCBOXi5) and 
I am receiving couple emails a day whenever ip's are banned, but I am in habit of installing ssmtp and bsd-mailx  and that install removes postfix.

**Comment by sonicwave2020** [comment]
I have checked fail2ban logs. 
I did see where my test device’s ip was banned when I purposely did a wrong NC login 6 times in a row. 
After 6 attempts it would not let me log in from that device’s ip as I was shown a blank web page.  After the allotted time, that ip was allowed again. 
I also did that test again & was able to unban a test device’s ip manually & that feature worked too. 
I saw fail2ban log entries for both of those tests. I did not see in fail2ban, where it attempted any email, but maybe it doesn’t show that there.

**Comment by nachoparker** [comment]
maybe try purging and reinstalling postfix? see if the aliases.db error goes away

**Comment by sonicwave2020** [comment]
Ok. Any tips on how to do that? Or would a quick web search be easy enough?

**Comment by nachoparker** [comment]
```
apt purge postfix
apt install postfix
```

**Comment by sonicwave2020** [comment]
postfix purged & then installed. will test later today

**Comment by sonicwave2020** [comment]
Tested by entering wrong password 6 times in a row on a device. Device appears to be banned, because it will not show login page now. 
I wasn’t sure what to put in prompts when doing postfix install. I may have entered wrong options. 
I chose option 2- I think it was “website plus” & put in my email provider’s domain, then changed the “NextCloudPi” to my email address’s user name. 
I went to NCP-config webUI & double checked that fail2ban is active & email is entered & checked to send mail. I can check logs later today.
